### ISSUE TITLE
Cannot redefine property: prepareStackTrace

### ISSUE THREAD

**Issue by hsdhillon** [Microsoft/ApplicationInsights-node.js#339]
I am trying to use AI with [loopback](https://github.com/strongloop/loopback) application but having trouble starting the application with AI setup. Below is the error trace. I am using nodejs 6.11 and loopback version 3.14. Has anyone encounter issue like this before?

```javascript
/home/hd/projects/sg/node_modules/applicationinsights/out/AutoCollection/Exceptions.js:33
                        throw error;
                        ^

/home/hd/projects/sg/node_modules/applicationinsights/out/AutoCollection/Exceptions.js:33
                        throw error;
                        ^

TypeError: Cannot redefine property: prepareStackTrace
    at Function.defineProperty (native)
    at Object.<anonymous> (/home/hd/projects/sg/node_modules/stack-chain/stack-chain.js:145:8)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at Module.patchedRequire [as require] (/home/hd/projects/sg/node_modules/diagnostic-channel/dist/src/patchRequire.js:14:46)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/hd/projects/sg/node_modules/stack-chain/index.js:15:41)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Comment by OsvaldoRosado** [comment]
Thanks for reporting this!

A few questions:
Which version of the applicationinsights package are you using? Is it your first import? Are you able to reproduce this with a small example of sample code?

Thanks!

**Comment by hsdhillon** [comment]
@OsvaldoRosado I found issue in my own code based on your pointer. Apparently, it wasn't the first require in my application.

Loopback uses server/server.js as first entry point to launch the application and that is where I placed applicationinsight initially. During troubleshooting I realized that to make loopback application run in Azure container, I added app.js which was calling server/server.js. All I had to do was move AI to app.js and things started working fine.

I appreciate your prompt response and help!
